app/views/staff/shared/xml/_activity.xml.haml

```diff
@@ -37,22 +37,22 @@
     %recipient-region{"code" => activity.recipient_region, vocabulary: "1"}
       %narrative= region_name_from_code(activity.recipient_region)
   %sector{"vocabulary" => "1", "code" => activity.sector}/
-  - unless activity.gender_not_assessed?
-    %policy-marker{"vocabulary" => "1", "code" => "1", "significance" => Activity.policy_marker_genders[activity.policy_marker_gender]}/
-  - unless activity.biodiversity_not_assessed?
-    %policy-marker{"vocabulary" => "1", "code" => "5", "significance" => Activity.policy_marker_biodiversities[activity.policy_marker_biodiversity]}/
-  - unless activity.climate_change_mitigation_not_assessed?
-    %policy-marker{"vocabulary" => "1", "code" => "6", "significance" => Activity.policy_marker_climate_change_mitigations[activity.policy_marker_climate_change_mitigation]}/
-  - unless activity.climate_change_adaptation_not_assessed?
-    %policy-marker{"vocabulary" => "1", "code" => "7", "significance" => Activity.policy_marker_climate_change_adaptations[activity.policy_marker_climate_change_adaptation]}/
-  - unless activity.desertification_not_assessed?
-    %policy-marker{"vocabulary" => "1", "code" => "8", "significance" => Activity.policy_marker_desertifications[activity.policy_marker_desertification]}/
-  - unless activity.disaster_risk_reduction_not_assessed?
-    %policy-marker{"vocabulary" => "1", "code" => "10", "significance" => Activity.policy_marker_disaster_risk_reductions[activity.policy_marker_disaster_risk_reduction]}/
-  - unless activity.disability_not_assessed?
-    %policy-marker{"vocabulary" => "1", "code" => "11", "significance" => Activity.policy_marker_disabilities[activity.policy_marker_disability]}/
-  - unless activity.nutrition_not_assessed?
-    %policy-marker{"vocabulary" => "1", "code" => "12", "significance" => Activity.policy_marker_nutritions[activity.policy_marker_nutrition]}/
+  - unless activity.policy_marker_gender == 1000
```

Comment:
Is there a way we could use a meaningful name instead of the numeral `1000` in these checks? I'm assuming it refers to the "not assessed" enum value.